### DISCUSSION TITLE
sveltekit and nextauth

### DISCUSSION THREAD

**Discussion by derheld42** [nextauthjs/next-auth#8824] [Help]
### Summary

I have these in my package.json file:
		"@auth/sveltekit": "^0.3.9",
		"@auth/core": "^0.16.1",
		"@auth/sequelize-adapter": "^1.0.3",

I started down the path of installing nextauth, then noticed that the latest sveltekit has a dependency on @auth/core.

So... will there be a sveltekit version that's compatible with nextauth?
I also noticed the example sveltekit auth uses @auth/core, so perhaps we're not there yet with a sveltekit nextauth integration?

thanx!

**Comment by tdack** [comment]
NextAuth is becoming AuthJS, I think eventually everything will end up referencing AuthJS. The documentation on https://authjs.dev isn't quite there yet, so you may find yourself referencing things on https://next-auth.js.org, particularly provider documentation.

The SvelteKit adapter works ok.

**Reply by jasongitmail** [reply]
@tdack Any chance you're using the Drizzle adapter with AuthJS and SvelteKit? 

I'm having trouble identify what to pass into `DrizzleAdapter(db)` because neither client nor db work:

```ts
// db.ts

import { createClient } from '@libsql/client';
import { drizzle } from 'drizzle-orm/libsql';
import * as env from '$env/static/private';

export const client = createClient({ authToken: env.TURSO_DB_AUTH_TOKEN, url: env.TURSO_DB_URL });

export const db = drizzle(client);
```

**Comment by tdack** [comment]
I'm not, but have you tried a schema as detailed at
https://authjs.dev/reference/adapter/drizzle ?

**Reply by jasongitmail** [reply]
yes, I have. That's alright, I started a [new discussion thread](https://github.com/nextauthjs/next-auth/discussions/8828) with full details.
